Documentation/spelling_wordlist.txt

```diff
@@ -250,6 +250,7 @@ kubernetes
 Kubernetes
 KV
 kvstore
+labelled
```

Comment:
We should also add `labeled` since we mostly use american spelling, I'm afraid. It might also be better to "fix" `labelled` so we're consistent (but I doubt it will be a problem in any case).